packages/frontend/src/hooks/useFeatureFlagEnabled.ts

```diff
@@ -1,9 +1,36 @@
-import { type FeatureFlags } from '@lightdash/common';
+import {
+    type ApiError,
+    type FeatureFlag,
+    type FeatureFlags,
+} from '@lightdash/common';
+import { useQuery } from '@tanstack/react-query';
 import { useFeatureFlagEnabled as useFeatureFlagEnabledPosthog } from 'posthog-js/react';
+import { lightdashApi } from '../api';
 
 /**
  * Thin wrapper around posthog's useFeatureFlagEnabled hook that is aware
  * of our FeatureFlags enum.
  */
 export const useFeatureFlagEnabled = (featureFlag: FeatureFlags) =>
     useFeatureFlagEnabledPosthog(featureFlag) === true;
+
+/**
+ * Use our own endpoint to get the feature flag from multiple sources.
```

Comment:
Should we add a comment mentioning that not all feature flags are implemented in the new endpoint? this way we don't risk people trying to use it for things that aren't there yet